src/components/SharedComponents/helpers/downloadAndShareHelpers/createDownloadResourcesLink.tsx

```diff
@@ -34,7 +35,16 @@ const createDownloadResourcesLink = async (
     selection,
     isLegacyDownload,
   };
-  const res = await fetch(downloadEnpoint);
+  const res = await fetch(
+    downloadEnpoint,
+    authToken
```

Comment:
The token has been threaded through to the underlying request to the curriculum downloads API